content/20.treatments.md

```diff
@@ -286,7 +286,32 @@ When these identifiers are followed back to the registry, most trials claim they
 For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
 At the very least, an inability to locate primary data means such claims cannot be verified.
 Additionally, citation of unavailable sources cast doubt on policies and research based on the assumption that HCQ is effective in treating COVID-19.
-
+A randomized, open label, non-placebo, trial of 150 COVID-19 patients was conducted in parallel in 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ [@doi:10.1101/2020.04.10.20060558].
+The trial examined the efficacy of HCQ in conjunction with standard of care (SOC) compared to SOC alone in 150 infected patients assigned randomly to the two respective groups (75 per group). 
+Most of the SARS-CoV-2 cases in this cohort were mild to moderate (98%), and the average age of the patients was 46 years.
+The primary end point of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups. 
+The secondary end points were amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C,SpO2 >94% on room air,and disappearance of symptoms like shortnes of breath, cough, sore throat.
+The median time to symptom alleviation was similar across conditions (19 days in HCQ+SOC vs 21 days in SOC). 
+However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of anti-virals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83.
+However, the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be under-powered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
+Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC alone group (mean of 0.062 versus 0.008; p=0.57). 
+Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response and subsequently prevent multiorgan failure and death. 
+Additionally, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes as opposed to 8.8% of patients receiving only SOC. 
+The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs 0% in the SOC group, p=0.004). 
+
+The study had a few limitations. 
+One of the limitations of the study was that the cohort mostly consisted of patients with mild/moderate disease symptoms, and the average patient in the analysis was 46.
+This may not be representative of COVID-19 patients broadly because older age groups are known to be at higher risk.
```

Comment:
```suggestion
Therefore, this sample may not be representative of COVID-19 patients broadly because older age groups are known to be at higher risk.
```
Building on @matfax suggestion